lib/createRef.lua

```diff
@@ -0,0 +1,20 @@
+--[[
+	Provides an API for acquiring a reference to a reified object. This
+	API is designed to mimic React 16.3's createRef API.
+
+	See:
+	* https://reactjs.org/docs/refs-and-the-dom.html
+	* https://reactjs.org/blog/2018/03/29/react-v-16-3.html#createref-api
+]]
+
+local refMetatable = {
+	__tostring = function(self)
+		return ("RoactReference(%q)"):format(tostring(self.current))
```

Comment:
Thinking about it now, do you think we should just format these with `%s` instead of `%q`? Since they'll mostly be Roblox objects, I don't know if it makes much sense to quote them!

Reply:
That's true - I'm used to quoting everything, but in this case it doesn't do much good! Fixed now.